setup.py

```diff
@@ -90,6 +91,10 @@ def get_requirements():
             'koji = rebasehelper.build_tools.koji_tool:KojiBuildTool',
             'copr = rebasehelper.build_tools.copr_tool:CoprBuildTool',
         ],
+        'rebasehelper.srpm_build_tools': [
+            'rpmbuild = rebasehelper.srpm_build_tools.srpm_rpmbuild_tool:RpmbuildSRPMBuildTool',
+            'mock = rebasehelper.srpm_build_tools.srpm_mock_tool:MockSRPMBuildTool',
```

Comment:
I think the package names should be just `rpmbuild_tool` and `mock_tool`, but if you want to keep `srpm` there, it should  be `rpmbuild_srpm_tool` and `mock_srpm_tool`.